package.json

```diff
@@ -39,11 +40,14 @@
     "zone.js": "^0.8.12"
   },
   "devDependencies": {
+    "@angular/bazel": "^5.0.1",
     "@angular/compiler-cli": "^5.0.0",
     "@angular/http": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/platform-server": "^5.0.0",
     "@angular/router": "^5.0.0",
+    "@angular/upgrade": "^5.0.1",
```

Comment:
any particular reason you added upgrade?

Reply:
Without it the router fails the `postinstall` ngc:
```
> ngc -p angular.tsconfig.json

node_modules/@angular/router/upgrade/src/upgrade.d.ts(9,31): error TS2307: Cannot find module '@angular/upgrade/static'.
```